packages/@react-spectrum/well/docs/Well.mdx

```diff
@@ -0,0 +1,52 @@
+<!-- Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. -->
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-spectrum/well';
+import {HeaderInfo, PropTable} from '@react-spectrum/docs';
+import packageData from '../package.json';
+
+```jsx import
+import {Well} from '@react-spectrum/well';
+```
+
+# Well
+
+<p>{docs.exports.Well.description}</p>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['Well']}
+  sourceData={[]} />
+
+## Example
+
+```tsx example
+<Well>Better a little which is well done, than a great deal imperfectly.</Well>
+```
+
+## Content
+
+Wells can have children. They can be nested.
+
+```tsx example
+<Well>Well</Well>
+
+<Well>Well <Well>oh well.</Well></Well>
+```
+
+### Internationalization
+
+In order to internationalize a well, a localized string should be passed to the `children` prop.
```

Comment:
Might want to add some of the accessibility section from v2 docs about the roles and such.

Reply:
@dannify updated; lmk if it islooks ok!